portality/migrate/20220928_1735_missing_admin/operations.py

```diff
@@ -0,0 +1,4 @@
+def add_admin_field(article):
+    if not "admin" in article.data:
+        article.data["admin"] = {"in_doaj": True}
```

Comment:
Is this always true?  I thought those items missing admin data were like that because the journal was withdrawn or missing?  Seems more likely that we should look the journal up and see what state it is in before setting this.